### DISCUSSION TITLE
Workspace build failure

### DISCUSSION THREAD

**Discussion by ryanmerolle** [coder/coder#3377] [General]
Hello,  I am testing coder in an partly air-gapped environment.  We leverage nexus for proxying apt, yum, and docker repos.

So far things setup with the docker install for testing.

I can connect to my repos just fine but when I build an workspace from an internal image, the workspace build process fails at step 7:
```
get dialer: negotiate rtc connection: error from peer: dial server [stun:stun.l.google.com:19302]: mismatched protocols
```

Any help would be greatly appreciated!

**Comment by kylecarbs** [comment]
Is this for v1 or v2 @ryanmerolle? Looks like this error is from v1.
